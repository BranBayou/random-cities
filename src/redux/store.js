import { configureStore } from '@reduxjs/toolkit';
import citiesReducer from './cities/citiesSlice';

const store = configureStore({
  reducer: {
    cities: citiesReducer,
  },
});

export default store;
