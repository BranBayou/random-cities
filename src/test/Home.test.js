import renderer from 'react-test-renderer';
import React from 'react';
import { Provider } from 'react-redux';
import store from '../redux/store';

test('should render Home', () => {
  const component = renderer.create(
    <Provider store={store} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('render city detail', () => {
  const componentDetail = renderer
    .create(<Provider store={store} />)
    .toJSON();
  expect(componentDetail).toMatchSnapshot();
});
