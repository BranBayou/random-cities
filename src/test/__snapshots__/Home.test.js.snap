// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render city detail 1`] = `null`;

exports[`should render Home 1`] = `null`;
